ocaml/testsuite/tests/typing-layouts/modules_beta.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_beta"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
378 - priorize `it's used as an element in a first-class module.` less than other reasons?